Index: src/pages/bookingSteps/SearchForm.jsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/pages/bookingSteps/SearchForm.jsx b/src/pages/bookingSteps/SearchForm.jsx
--- a/src/pages/bookingSteps/SearchForm.jsx	
+++ b/src/pages/bookingSteps/SearchForm.jsx	(date 1744204945493)
@@ -86,7 +86,7 @@
                             </IconButton>
                         </Grid>
 
-                        <Grid size={4}>
+                        <Grid size={3.5}>
                             <FormInput
                                 name="destination"
                                 label="To *"
